packages/cli/src/config/auto-init/ApimlAutoInitHandler.ts

```diff
@@ -73,12 +73,17 @@ export default class ApimlAutoInitHandler extends BaseAutoInitHandler {
         if ((session.ISession.user && session.ISession.password) || (session.ISession.cert && session.ISession.certKey)) {
             // If it is basic authentication, we need to set the auth type.
             if (session.ISession.password) { session.ISession.type = SessConstants.AUTH_TYPE_TOKEN; }
-            session.ISession.tokenType = SessConstants.TOKEN_TYPE_APIML;
+            if (session.ISession.tokenType == null) {
```

Comment:
What if the tokenType is not APIML?

Reply:
The `zowe config auto-init` command should only work with APIML tokens. I don't believe there should be an instance where the tokenType should be different. It could be dynamic, in which case it will get replaced after the login operation is performed further down.

Reply:
I was just wondering in case something else (unlikely) was setting up tokenType in the base profile as something else, and getting caught up with that value during the config load. Just an edge case, not sure if it makes sense in this context.